src/QGCMessageBox.h

```diff
@@ -115,6 +115,10 @@ private slots:
             QMessageBox box(icon, emptyTitle, title, buttons, parent);
             box.setDefaultButton(defaultButton);
             box.setInformativeText(text);
+            // Get this thing off a proper size
+            QSpacerItem* horizontalSpacer = new QSpacerItem(500, 0, QSizePolicy::Minimum, QSizePolicy::Expanding);
+            QGridLayout* layout = (QGridLayout*)box.layout();
+            layout->addItem(horizontalSpacer, layout->rowCount(), 0, 1, layout->columnCount());
```

Comment:
QMessageBox sizing, particularly the width, under Mac OS is awful. It's always too narrow and breaks the text down to many lines. What I did above was to force to be at least 500 pixels wide.